src/web/components/Nav/ExpandedMenu/ExpandedMenuToggle/VeggieBurger.tsx

```diff
@@ -20,12 +16,10 @@ const veggieBurger = ({
     border: 0;
     border-radius: 50%;
     outline: none;
-    ${until.tablet} {
-        z-index: 1;
-    }
-    ${from.tablet} {
-        z-index: ${showExpandedMenu ? 1071 : 0};
-    }
+
+    /* TODO: we should not use such a hight z-index number  */
+    z-index: 1071;
```

Comment:
Shall we try changing this as part of this PR? Boy scouting and all that

Reply:
this may open up a can of worms later, I prefer to do this outside of this PR as it is already quite large, there are several areas where we have a hard coded `z-index`.

![Screenshot 2020-04-27 at 11 15 35](https://user-images.githubusercontent.com/8831403/80361399-c291a780-8878-11ea-9f99-14ba69ea5195.png)

Also I plan on doing some fixes to veggie burger and Top Nav Ad z-index, I can expand the scope of these tickets?

https://trello.com/c/y8CyFKJm/1524-top-nav-ad-and-nav-z-index-issue
https://trello.com/c/cgzbjP6u/1525-veggie-burger-z-index

Reply:
Should maybe be handled by https://github.com/guardian/dotcom-rendering/blob/master/src/web/lib/getZIndex.tsx

Reply:
Fair enough 👍